lmdeploy/serve/async_engine.py

```diff
@@ -247,19 +247,28 @@ def safe_run(self, session_id: Optional[int] = None):
 
     async def get_generator(self, stop: bool, session_id: int):
         """Only return the model instance if it is available."""
-        if stop:
-            return self.engine.create_instance()
-        while self.gens_set == set():
-            await asyncio.sleep(0)
-        if str(session_id) in self.id2generator:
```

Comment:
Just append `and self.backend == 'pytorch` please.

Reply:
> Just append `and self.backend == 'pytorch` please.

ok

Reply:
> Just append `and self.backend == 'pytorch` please.

done